adr/0015-carrying-over-applications.md

```diff
@@ -0,0 +1,81 @@
+# 15. Carrying over applications
+
+Date: 2020-08-12
+
+## Status
+
+Accepted
+
+## Context
+
+The current recruitment cycle ends on 18th September 2020. At that point there
+will be some candidates who could benefit from their application being carried
+over to the next cycle. Carrying over an application means the candidate can
+apply to courses in the new recruitment cycle without having to fill in the
+whole application form again.
+
+### Carrying over an application makes sense in the following states
+
+#### Before the application reaches the provider
+
+- Withdrawn
+- Unsubmitted
+- Ready to send to provider
+
+#### After the application can’t progress any further
+
+- Conditions met (recruited)
```

Comment:
`Conditions met (recruited)` appears both here and in the 'does not make sense' section. Which one should stay?